### PR TITLE
Move DelegateFactory to Delegate package

### PR DESCRIPTION
**Pull request by dmosites**

### CODE REVIEW

File: source/delegate-factory/deploys.json [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
-  "1": "0xA55523eeD98Ba64fF7673b22963DF34E3A0305FE",
```
Comment (alicevhenshaw) [inline source/delegate-factory/deploys.json]: do we want to keep these in the delegate? Seems like something we shouldnt lose

File: scripts/slither.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@
 # ------------------------------------------------------------------
 
 
- array=( indexer index delegate delegate-factory swap types wrapper )
+ array=( indexer index delegate swap types wrapper )
```
Comment (alicevhenshaw) [inline scripts/slither.sh]: oh. Index is still here.

Comment (dmosites) [reply scripts/slither.sh]: Noticed that along the way. Updated.

File: source/delegate-factory/deploys.json (hunk 1 above)
Comment (dmosites) [reply source/delegate-factory/deploys.json]: That's right, good call. Though this file should indicate what is actually deployed. To discuss

Comment (dmosites) [reply source/delegate-factory/deploys.json]: Added some README content to address.

**Pull request merged**